Gemfile

```diff
@@ -41,7 +41,7 @@ end
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 # Slack Ruby Bot Server
-gem 'slack-ruby-bot-server'
+gem 'slack-ruby-bot-server', github: 'dblock/slack-ruby-bot-server', branch: 'oauth-v2'
 gem 'kaminari-mongoid'
```

Comment:
I released 1.1.0 so you can take a dependency on that.

Reply:
Thanks a lot for informing!